device-registry/src/main/scala/org/genivi/sota/device_registry/db/InstalledPackages.scala

```diff
@@ -53,6 +53,9 @@ object InstalledPackages {
       installedPackages ++= packages.map(InstalledPackage(device, _, Instant.now()))
     ).transactionally
 
+  def addRows(rows: Seq[InstalledPackage])(implicit ec: ExecutionContext): DBIO[Seq[Int]] =
+    DBIO.sequence(rows.map(installedPackages += _)).transactionally
```

Comment:
Yeah, I envisioned it either adding rows or failing due to duplicate keys. How is that not idempotent? Sure, it might error if we run it twice, but that wouldn't be disastrous.